release/release_tests.yaml

```diff
@@ -633,7 +633,7 @@
     cluster_compute: compute_cpu_16.yaml
 
   run:
-    timeout: 1800
+    timeout: 3600
```

Comment:
How long was this taking in your runs? An hour seems like a lot for this test.

Reply:
~40 minutes total for all three dataset sizes. Yeah I think we should try to reduce the time of all of the tests, but that might be better to do as one pass once all of the benchmarks are in.